30-indoor-air-quality-and-gas-leak/firmware/src/main.cpp

```diff
@@ -0,0 +1,278 @@
+#include <Arduino.h>
+#include <Notecard.h>
+#include "SparkFun_ENS160.h"
+
+#include "STM32LowPower.h"
+
+/**
+ * @brief Collects together the individual readings from the gas sensor.
+ */
+struct AQISensorReading {
+    uint16_t tvoc;
+    uint16_t eco2;
+    uint16_t etoh;
+    uint8_t aqi;
+
+    /**
+     * @brief Set to true when the values measured should signal an alert.
+     */
+    bool alert;
+    float temp;
+    float rh;
+
+    inline bool isAlert() const {
+        return alert;
+    }
+};
+
+
+bool isAlert(AQISensorReading& reading);
+static bool registerNotefileTemplate();
+static void addNote(const AQISensorReading& reading, bool immediate, uint8_t alert);
+
+
+Notecard notecard;
+#define productUID "com.your-company.your-name:your_product"
```

Comment:
Can you please switch this to this approach? https://github.com/blues/app-accelerators/blob/main/09-valve-monitor/firmware/arduino/src/main.cpp#L32-L39 We like to force people to change this value or get an error.